Sources/CustomHTTPProtocol.swift

```diff
@@ -42,6 +42,11 @@ public class CustomHTTPProtocol: URLProtocol {
         if CustomHTTPProtocol.property(forKey: Constants.RequestHandledKey, in: request) != nil {
             return false
         }
+        
+        if request.url?.scheme == "wss" || request.url?.scheme == "ws" {
```

Comment:
Could we have a static property similar to `ignoredHosts` that could be configured instead?

We've recently run into this issue whilst trying to use Starscream with native WebSocket APIs and we wouldn't necessarily be interested in logging any of the WebSocket traffic, but this would be our choice only; it would feel like a better approach rather than hardcode these exceptions like what is done here.